### DISCUSSION TITLE
Custom Password Within Advanced

### DISCUSSION THREAD

**Discussion by CameronWiller** [cupcakearmy/cryptgeon#60] [Ideas]
It would be great to be able to set a custom password when creating the note.
Apologies if this is already possible, if this is the case is there a guide on configuring this at all?

**Comment by cupcakearmy** [comment]
It used to be a thing but i was removed. What would be the use case? (just curious)

**Reply by CameronWiller** [reply]
We are looking at distributing secure notes locked behind a code that we then send via SMS to the end user regarding passwords for new accounts.

It essentially would allow for a "2FA Lite" by sending the user the link to the note, then requiring a secondary form of authentication to open the note (SMS Code) 

If it's been removed, that's understandable, just an extra bit of functionality that would fit our specific use case :)

Thanks for the prompt reply

**Reply by cupcakearmy** [reply]
Interesting use case. Maybe in the future, but unfortunately I don't have time rn. PRs are always welcomed otherwise :)
